models/tokens/ethereum/tokens_ethereum_transfers_base.sql

```diff
@@ -0,0 +1,17 @@
+{{ config(
+        alias = alias('transfers_base'),
+        tags=['dunesql'],
+        partition_by = ['block_date'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['block_number', 'tx_index', 'evt_index', 'trace_address'],
+        )
+}}
+
+{{transfers_base(
+    blockchain='ethereum',
+    traces = source('ethereum','traces'),
+    transactions = source('ethereum','transactions'),
+    erc20_transfers = source('erc20_ethereum','evt_Transfer'),
+)}}
```

Comment:
I think we'd want to include the WETH deposit and withdraw events here. 
It's a bit of an ethereum specific edge case so I don't think we should add it to the macro. We could union it in here?

Reply:
I've included it the macro now. It exists for polygon as well at least. It's optional in the macro though.